packages/client/src/v2-events/components/DownloadButton.tsx

```diff
@@ -139,7 +139,7 @@ export function DownloadButton({
     eventDocument.isFetched
 
   if (isDraft && isDownloadedToMe) {
-    return <Downloaded />
+    return <Downloaded data-testid="downloaded-icon" />
```

Comment:
can we end up rendering same data-testid's on the same page? does it matter?

Reply:
Probably shouldn't matter even if that happens. Won't happen now since this component is not used on the same page twice.